CRM/Mailing/BAO/Mailing.php

```diff
@@ -2637,22 +2637,8 @@ public function getReturnProperties() {
    */
   public static function commonCompose(&$form) {
     //get the tokens.
-    $tokens = CRM_Core_SelectValues::contactTokens();
+    $tokens = array();
 
-    $className = CRM_Utils_System::getClassName($form);
-    if ($className == 'CRM_Mailing_Form_Upload') {
-      $tokens = array_merge(CRM_Core_SelectValues::mailingTokens(), $tokens);
-    }
-    elseif ($className == 'CRM_Admin_Form_ScheduleReminders') {
-      $tokens = array_merge(CRM_Core_SelectValues::activityTokens(), $tokens);
-      $tokens = array_merge(CRM_Core_SelectValues::eventTokens(), $tokens);
-      $tokens = array_merge(CRM_Core_SelectValues::membershipTokens(), $tokens);
-    }
-    elseif ($className == 'CRM_Event_Form_ManageEvent_ScheduleReminders') {
-      $tokens = array_merge(CRM_Core_SelectValues::eventTokens(), $tokens);
-    }
-
```

Comment:
Thanks for the review @davecivicrm 
When checking the current behavior on master it seems all tokens (activity / membership / event) are added to the schedule reminder form regardless if it's called from the event UI or from the admin/communication/scheduled reminders UI. 
So this seems to keep the current behaviour. 
If i see the current code correctly the CRM_Event_Form_ManageEvent_ScheduleReminders form adds indeed only event tokens, however this form doesn't use tokens. 
The git log tells this form was cleaned up in https://issues.civicrm.org/jira/browse/CRM-15291 and https://github.com/civicrm/civicrm-core/pull/4644 and it seems this code was a leftover which could be removed there.